README.md

```diff
@@ -106,6 +106,8 @@ Override some of the conventions:
 
 ```ruby
 Parity.configure do |config|
```

Comment:
While we're touching the README - where would this configuration go? Where does Parity look for configuration?